disnake/emoji.py

```diff
@@ -103,9 +103,7 @@ def _from_data(self, emoji: EmojiPayload):
         self.user: Optional[User] = User(state=self._state, data=user) if user else None
 
     def _to_partial(self) -> PartialEmoji:
-        emoji = PartialEmoji(name=self.name, animated=self.animated, id=self.id)
-        emoji._state = self._state
-        return emoji
+        return PartialEmoji(name=self.name, animated=self.animated, id=self.id)
```

Comment:
why does this no longer have state attached?

Reply:
That's the original behavior - I attached the state when forum tags used `emoji._to_partial()` so that you could still use `await tag.emoji.read()`, but removed the `._to_partial` call again in 1d5aeb4e16e480ded5d3422ea9d2f034c90f41dd, which made this change unnecessary.

Reply:
Why don't we provide state?

Reply:
It's only used for components, which always use partial emojis and are completely detached from the connection state.